src/covid_model_seiir_pipeline/workflow_template.py

```diff
@@ -2,50 +2,128 @@
 from typing import Dict
 
 from jobmon.client import Workflow, BashTask
-from jobmon.client.swarm.executors.base import ExecutorParameters
+from jobmon.client.swarm.executors.base import ExecutorParameters as ExecutorParameters_
 from jobmon.client.swarm.workflow.task_dag import DagExecutionStatus
 
 from covid_model_seiir_pipeline import utilities
 
+DEFAULT_PROJECT = 'proj_dq'
+DEFAULT_QUEUE = 'd.q'
+
+
+class ExecutorParameters(ExecutorParameters_):
+
+    def __init__(self,
+                 queue: str,
+                 max_runtime_seconds: int = None,
+                 m_mem_free: str = None,
+                 num_cores: int = None,
+                 **kwargs):
+        if max_runtime_seconds is None:
+            max_runtime_seconds = self.default_runtime
+        if m_mem_free is None:
+            m_mem_free = self.default_memory
+        if num_cores is None:
+            num_cores = self.default_cores
+        super().__init__(num_cores=num_cores,
+                         queue=queue,
+                         max_runtime_seconds=max_runtime_seconds,
+                         m_mem_free=m_mem_free,
+                         **kwargs)
+
+    @abc.abstractmethod
+    @property
+    def default_runtime(self) -> int:
+        ...
```

Comment:
Fun fact: this line would be valid PERL, and it would do the equivalent of `raise NotImplementedError()` if it was ever run by the interpreter